src/main/java/org/opensearch/jobscheduler/JobSchedulerPlugin.java

```diff
@@ -151,6 +152,10 @@ public List<ExecutorBuilder<?>> getExecutorBuilders(Settings settings) {
 
     @Override
     public void onIndexModule(IndexModule indexModule) {
+        if (indexModule.getIndex().getName().equals(JobDetailsService.JOB_DETAILS_INDEX_NAME)) {
+            indexModule.addIndexOperationListener(this.jobDetailsService);
+            log.info("JobDetailsService started listening to operations on index {}", indexModule.getIndex().getName());
```

Comment:
This works but since we already checked equality wouldn't it be simpler to use `JobDetailsService.JOB_DETAILS_INDEX_NAME` here?